src/polycubed/src/polycubed.cpp

```diff
@@ -278,6 +278,9 @@ int main(int argc, char *argv[]) {
   // register services that are shipped with polycube
   load_services(*core);
 
+  // load the last topology created
+  restserver->load_last_topology();
```

Comment:
Given the previous comment, if the user asked to start with an empty topology, we don't have to load the previous topology here.